<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2">
     <widget class="QTabWidget" name="CtrlPan">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Img Info</string>
       </attribute>
       <widget class="QSplitter" name="splitter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>220</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="loadImgRef">
         <property name="text">
          <string>Load Img Ref</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadImgMtch">
         <property name="text">
          <string>Load Img Mtch</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="exeDeepMtch">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>70</y>
          <width>111</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Deep Matching</string>
        </property>
       </widget>
       <widget class="QPushButton" name="help">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>570</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>help</string>
        </property>
       </widget>
       <widget class="QPushButton" name="configDM">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>config. DM</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="imgRefDisp">
      <property name="text">
       <string>Reference Image</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="imgMtchDisp">
      <property name="text">
       <string>Match Image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
